content/working-groups/server/extended-validation/certificate-contents.md

```diff
@@ -1,32 +1,31 @@
 ---
 aliases:
 - /ev-certificate-contents/
-date: 2013-09-04 03:28:58
+date: 2024-04-02 18:28:58
 title: EV Certificate Contents
 ---
 
-**EV SSL Certificate Contents**
+**EV TLS Certificate Contents**
 
-An EV SSL Certificate authenticates a web site (by domain name) and the entity controlling the web site. An EV SSL Certificate contains the following required fields:
+An EV TLS Certificate authenticates a web site (by domain name) and the entity controlling the web site. An EV TLS Certificate contains the following required fields:
 
-**Subject Identity**
+1. **Subject Organization name** (Old EVG 9.2.1 - New EVG 7.1.4.2.1) – The “O” field must contain the full legal name of the entity controlling the web site as listed in the official records in the Subject’s Jurisdiction or as otherwise verified by the CA according to the EV Guidelines (EVG).
```

Comment:
We don't have 7.1.4.2.1 yet in the EVG until April 15 when SC-65 clears the IPR review period. I propose we remove the section reference. The same applies for the remaining changes that point to the new section 7.1.

Reply:
Or do we just hold this one until April 15th?

Reply:
Is this ready to be merged?